examples/serial/serial.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2017 Kyle Robbertze <krobbertze@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include <delay.h>
+#include <fx2macros.h>
+#include <fx2regs.h>
+#include <fx2types.h>
+
+#define PD3 0xB3
+#define BAUD 32
+
+__sbit __at PD3 USART; // USART slave send from port D3
+
+void usart_init(void) {
+    SETCPUFREQ(CLK_48M);
+	USART = 1;
+	OED |= 0xff;
```

Comment:
This I pulled from the serial.c file in the main firmware. I do not know what exactly it does. I assume it initialises something useful